src/cockpit/beiboot.py

```diff
@@ -480,9 +509,10 @@ def main() -> None:
     polyfills.install()
 
     parser = argparse.ArgumentParser(description='cockpit-bridge is run automatically inside of a Cockpit session.')
-    parser.add_argument('--remote-bridge', choices=['auto', 'never', 'always'], default='auto',
+    parser.add_argument('--remote-bridge', choices=['auto', 'never', 'supported', 'always'], default='auto',
                         help="How to run cockpit-bridge from the remote host: auto: if installed (default), "
                         "never: always copy the local one; "
+                        "supported: copy local one for compatible OSes, fail otherwise; "
```

Comment:
I think `supported` will use the installed cockpit-bridge if there is one, right? This isn't clear from the docs here, they make it sound like it's a variant of `never`.

Reply:
Right, it prefers the remote bridge if installed. I fixed the help. Thanks for spotting!